recipes-kernel/linux/files/patches-5.10/0122-scripts-dtc-Update-to-upstream-version-v1.6.0-51-g18.patch

```diff
@@ -0,0 +1,993 @@
+From 5169b7666bfb0be113cf08c736a7395190e17689 Mon Sep 17 00:00:00 2001
+From: Rob Herring <robh@kernel.org>
+Date: Wed, 3 Feb 2021 15:26:03 -0600
+Subject: [PATCH 122/123] scripts/dtc: Update to upstream version
+ v1.6.0-51-g183df9e9c2b9
+
+This adds the following commits from upstream:
+
+183df9e9c2b9 gitignore: Ignore the swp files
+0db6d09584e1 gitignore: Add cscope files
+307afa1a7be8 Update Jon Loeliger's email
+ca16a723fa9d fdtdump: Fix gcc11 warning
+64990a272e8f srcpos: increase MAX_SRCFILE_DEPTH
+163f0469bf2e dtc: Allow overlays to have .dtbo extension
+3b01518e688d Set last_comp_version correctly in new dtb and fix potential version issues in fdt_open_into
+f7e5737f26aa tests: Fix overlay_overlay_nosugar test case
+7cd5d5fe43d5 libfdt: Tweak description of assume-aligned load helpers
+a7c404099349 libfdt: Internally perform potentially unaligned loads
+bab85e48a6f4 meson: increase default timeout for tests
+f8b46098824d meson: do not assume python is installed, skip tests
+30a56bce4f0b meson: fix -Wall warning
+5e735860c478 libfdt: Check for 8-byte address alignment in fdt_ro_probe_()
+67849a327927 build-sys: add meson build
+05874d08212d pylibfdt: allow build out of tree
+3bc3a6b9fe0c dtc: Fix signedness comparisons warnings: Wrap (-1)
+e1147b159e92 dtc: Fix signedness comparisons warnings: change types
+04cf1fdc0fcf convert-dtsv0: Fix signedness comparisons warning
+b30013edb878 libfdt: Fix kernel-doc comments
+
+Signed-off-by: Rob Herring <robh@kernel.org>
```

Comment:
This is a cherry-pick of https://github.com/torvalds/linux/commit/79edff12060fe7772af08607eff50c0e2486c5ba?